src/vellum/workflows/nodes/displayable/bases/base_prompt_node/node.py

```diff
@@ -38,6 +42,7 @@ def run(self) -> Iterator[BaseOutput]:
 
     def _process_prompt_event_stream(self) -> Generator[BaseOutput, None, Optional[List[PromptOutput]]]:
         try:
+            self._validate()
```

Comment:
[nit] no need to have this nested within the try/except, which we should usually try to keep as targeted as possible